docs/snippets/snippets.toml

```diff
@@ -234,6 +231,10 @@ quick_start = [ # These examples don't have exactly the same implementation.
 "archetypes/points3d_partial_updates" = [ # TODO(cmc): revert once C++ has tagged partial updates
   "cpp",
 ]
+"archetypes/points3d_send_columns" = [
+  "py", # TODO(#8752): needs tagged columnar APIs
+  "cpp", # TODO(#8754): needs tagged columnar APIs
+]
```

Comment:
isn't the result already/still the same? How are they differing now?

Reply:
Untagged in py and cpp